src/ssh/cockpitsshrelay.c

```diff
@@ -458,7 +459,15 @@ set_knownhosts_file (CockpitSshData *data,
         }
     }
 
-  /* TODO: Check more sources of known hosts here if !host_known */
+  /* check ~/.ssh/known_hosts, unless we are running as a system user ($HOME == "/"); this is
```

Comment:
I'd suggest moving this to cockpitrelay. The point of this file was more dealing with env vars and being able to set and read them from one place. This doesn't seem to fit that so well.